README.md

```diff
@@ -17,6 +17,15 @@ SyndiBox is meant to be an easy and stress-free way of implementing dialog into
  4. Fill the properties in the Inspector with your dialog, auto advance, font, text voice, color, and text speed. You can use what is filled in the image below as an example.![Fill the Inspector properties.](https://imgur.com/0POjPSz.png)
  5. Press the `Play Scene` button (or `F6` on your keyboard) and watch it print!![There it is~](https://imgur.com/Fiigoty.png)
 
+## Script Guide
+Syndibox exposes some functions for use within GDScript that allow for dynamic control of the text and effects.   
+
+start(string: dialog, int: start_position) - Resets, shows, and starts the dialog box, displaying the given string. Passing a string is functionally the same as if you entered the string as Dialog in the inspector. dialog defaults to ''. Start position indicates where to beging displaying the text. start_position defaults to 0.  
```

Comment:
It should be noted that leaving the dialog value blank just reprints the current dialog set

Reply:
Ah another good thing to be documented! I'll edit this.